### ISSUE TITLE
Override configuration is not applied while using run-workers command

### ISSUE THREAD

**Issue by KMKoushik** [codeceptjs/CodeceptJS#2109]
#### What are you trying to achieve?
Used `codeceptjs run-workers 2 -o '{"helpers": { "Puppeteer": { "url": "http://localhost", "show": false }}}'` To override show option.
#### What do you get instead?
Option is not applied while running test

### Details

* CodeceptJS version: 2.3.6
* NodeJS Version: 12.13.0
* Operating System: Linux, Ubuntu distro
* testcafe
* Configuration file:

```js
{
  tests: './test/*test.js',
  output: './output',
  helpers: {
    Puppeteer: {
      url: 'http://localhost',
      show: true
    }
  },
  include: {},
  bootstrap: null,
  mocha: {},
  name: 'codepup'
}
```

**Comment by koushikmohan1996** [comment]
Findings: 
While running run-worker js override config is loaded [here](https://github.com/Codeception/CodeceptJS/blob/master/lib/command/run-workers.js#L43). But it is not passed to the worker thread. Override config should also be passed along with options in worker data.

**Comment by neutraali** [comment]
Seems to still be a thing. Disappointing that you need to hard-wire a config for every case if you want to `run-workers`.

**Comment by neutraali** [comment]
... Sort of works? Using `codeceptjs 3.0.0-beta.4`

My override parameter is fine as JSON @ `codecept.conf.js`, but as it gets passed to workers something goes haywire?

```
(node:200) UnhandledPromiseRejectionWarning: SyntaxError: Unexpected token p in JSON at position 3
    at JSON.parse (<anonymous>)
    at WorkerObject.addConfig (/Selenium/node_modules/codeceptjs/lib/workers.js:130:28)
    at Workers.overrideConfig (/Selenium/node_modules/codeceptjs/lib/workers.js:277:14)
    at Command.module.exports (/Selenium/node_modules/codeceptjs/lib/command/run-workers.js:33:11)
    at Command.listener (/Selenium/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:314:20)
    at Command.parseArgs (/Selenium/node_modules/commander/index.js:651:12)
    at Command.parse (/Selenium/node_modules/commander/index.js:474:21)
    at Object.<anonymous> (/Selenium/node_modules/codeceptjs/bin/codecept.js:188:9)
    at Module._compile (internal/modules/cjs/loader.js:1076:30)
```

**Comment by koushikmohan1996** [comment]
@neutraali I will check this. I fixed this in 3.0. Meanwhile can you share with me the override config you just used?

**Comment by neutraali** [comment]
Hey @koushikmohan1996 - Glad someone's on this.

My override, when logged @ `codecept.conf.js` through `yargs`, looks something like this:

`'{"port":8181,"tests":"/path/to/test.js","ci":false}'`

If you want a `printf %q` -escaped shell version, here you go:

`-o \"\{\"port\":8181\,\"tests\":\"/path/to/test.js\"\,\"ci\":false\}\"`
